.travis.yml

```diff
@@ -108,3 +108,32 @@ script:
     else
       ./travis/run-large-python-tests.sh;
     fi
+
+notifications:
+  # Send Slack notifications on build being broken or fixed, and email (Pagerduty) notifications
+  # on build being broken. See
+  # https://docs.travis-ci.com/user/notifications/#changing-notification-frequency for more info.
+  email:
+    recipients:
+    - secure: "UFN6LLWRASxILSKY/hD5ugxP78Bhuq58X4H0ar6BnNl6SS70bhJFbJFuvkWBoLHAhydV876h2nBZLfTUCvf3\
```

Comment:
Use double-quotes & backslashes to allow newlines in YAML value without them being converted to spaces, as per https://yaml-multiline.info/